src/components/Hub/ImageDetails.tsx

```diff
@@ -21,41 +29,65 @@ const Platform = styled(Tag)`
   background-color: ${(props) => props.theme.palette.tagPlatformBackground};
 `;
 
+const getUsageInFlow = (dockerName: string, version: string) =>
+  `
+from jina.flow import Flow
+f = Flow().add(uses='${dockerName}:${version}')
+`
+const getPythonUsage = (dockerName: string, version: string) =>
+  `
+!Flow
+pods:
+  my_pod1:
+    image: "${dockerName}:${version}"
+`
+
 export default function ImageDetails({ image }: Props) {
-  let { keywords, platform, author } = image;
+  let { keywords, platform, author, version } = image;
   let dockerCommand = image["docker-command"];
+  let dockerName = image["docker-name"];
   return (
-    <Card className="readme-container mb-4">
-      <CardBody className="pt-0 pb-2">
-        <DetailsSection>
-          <SectionHeading>Tags</SectionHeading>
-          <div>
-            {keywords.map((keyword) => (
-              <Tag data-name="hubImageTags" key={keyword}>
-                {keyword}
-              </Tag>
-            ))}
-          </div>
-        </DetailsSection>
-        <DetailsSection>
-          <SectionHeading>Platform</SectionHeading>
-          <div>
-            {platform.map((p) => (
-              <Platform data-name="hubImageTags" key={p}>
-                {p}
-              </Platform>
-            ))}
-          </div>
-        </DetailsSection>
-        <DetailsSection>
-          <SectionHeading>Docker Pull Command</SectionHeading>
-          <div>{dockerCommand}</div>
-        </DetailsSection>
-        <DetailsSection>
-          <SectionHeading>Owner</SectionHeading>
-          <div>{author}</div>
-        </DetailsSection>
-      </CardBody>
-    </Card>
+    <>
+      <DetailCard>
+        <CardContent>
+          <DetailsSection>
+            <SectionHeading>Docker Command</SectionHeading>
+            <CodeSnippetWithCopy codeSnippet={dockerCommand} />
+            <SectionHeading>Python Usage</SectionHeading>
+            <CodeSnippetWithCopy codeSnippet={getPythonUsage(dockerName, version)} />
+            <SectionHeading>In a Flow</SectionHeading>
+            <CodeSnippetWithCopy codeSnippet={getUsageInFlow(dockerName, version)} />
+          </DetailsSection>
+        </CardContent>
+      </DetailCard>
+      <DetailCard>
+        <CardContent>
+          <DetailsSection>
+            <SectionHeading>Tags</SectionHeading>
+            <div>
+              {keywords.map((keyword) => (
+                <Tag data-name="hubImageTags" key={keyword}>
+                  {keyword}
+                </Tag>
+              ))}
+            </div>
+          </DetailsSection>
+          <DetailsSection>
+            <SectionHeading>Platform</SectionHeading>
+            <div>
+              {platform.map((p) => (
+                <Platform data-name="hubImageTags" key={p}>
```

Comment:
Maybe add a cypress test? Also each element would have the same data-name. Something like data-name={`hubImageTag-${index}`} would avoid that